lib/challenge.js

```diff
@@ -60,7 +76,11 @@ var some_pet;
 
 var Pet;
 // Answer Starts Here
-
+var Pet = function(name, species, breed, noise)
```

Comment:
This function is missing the curly braces that denote its body. How would anyone be able to tell which statements belong to the function, and which ones are outside of it?